.tool-versions

```diff
@@ -1,5 +1,5 @@
-python 3.7.12
-poetry 1.2.0
+python 3.10.12
+poetry 1.5.1
```

Comment:
Is there a reason why we are not upgrading directly to the latest 1.7.1?

Reply:
no reason. i did upgrade locally myself just before xmas

Reply:
done